packages/redux/src/exchanges/__tests__/reducer.test.ts

```diff
@@ -0,0 +1,217 @@
+import * as actionTypes from '../actionTypes.js';
+import {
+  Comparator,
+  Criteria,
+  ExchangeBookRequestStatus,
+  LogicOperatorType,
+  toBlackoutError,
+} from '@farfetch/blackout-client';
+import {
+  exchangeReturnItemId,
+  mockState,
+  orderId,
+  orderItemUuid,
+  returnId,
+} from 'tests/__fixtures__/exchanges/index.mjs';
+import { LOGOUT_SUCCESS } from '../../users/authentication/actionTypes.js';
+import reducer, * as fromReducer from '../reducer.js';
+import type { ExchangesState } from '../types/index.js';
+
+let initialState: ExchangesState;
+const randomAction = { type: 'this_is_a_random_action' };
+
+describe('exchanges reducer', () => {
+  beforeEach(() => {
+    initialState = fromReducer.INITIAL_STATE;
+  });
+
+  describe('reset handling', () => {
+    it.each([actionTypes.RESET_EXCHANGES, LOGOUT_SUCCESS])(
+      'should return initial state on %s action',
+      actionType => {
+        expect(
+          reducer(mockState.exchanges, { type: actionType, payload: {} }),
+        ).toMatchObject(initialState);
+      },
+    );
+  });
+
+  describe('exchangeFilters() reducer', () => {
+    it('should return the initial state', () => {
+      const state = reducer(undefined, randomAction).exchangeFilters;
+
+      expect(state).toEqual(initialState.exchangeFilters);
+      expect(state).toEqual({ error: null, isLoading: false, result: null });
+    });
+
+    it('should handle CREATE_EXCHANGE_FILTERS_REQUEST action type', () => {
+      expect(
+        reducer(undefined, {
```

Comment:
Please, use a state here that is more adequate for this test to catch errors. To avoid being exhaustive, at least test with a state that has the `isLoading` equals to `false` and either `error` or `result` have a value.

The same for the remaining tests that do not test for the return of the initial state.

Reply:
Fixed 👍